mailbag/helper.py

```diff
@@ -76,6 +83,40 @@ def moveWithDirectoryStructure(dry_run, mainPath, mailbag_name, input, emailFold
 
         return file_new_path
 
+
 def saveAttachments(part):
-    return (part.get_filename(),part.get_payload(decode=True))
-    
\ No newline at end of file
+    return (part.get_filename(), part.get_payload(decode=True))
+
+
+def saveFile(filePath, text):
+    with open(filePath, 'w') as f:
+        f.write(text)
+
+
+def deleteFile(filePath): 
+    if os.path.exists(filePath):
+        os.remove(filePath)
+
+
+def startServer(port=5000):
+    try:
+        Handler = http.server.SimpleHTTPRequestHandler
+        with socketserver.TCPServer(("", port), Handler) as httpd:
+            httpd.serve_forever()
+    # Stopped the server
+    except KeyboardInterrupt:
+        httpd.server_close()
+        log.debug("The server is stopped.")
+
+                        
+def saveWARC(dry_run, warc_dir, message, port=5000):
+    message_warc_dir = os.path.join(warc_dir, str(message.Mailbag_Message_ID))
+    filename = os.path.join(message_warc_dir, str(message.Mailbag_Message_ID) + ".warc.gz")
+    log.debug('Saving WARC:' + filename)
+    
+    if not dry_run:
+        os.mkdir(message_warc_dir)
```

Comment:
I think this might break at some point. Haven't tested but this might be what you need: https://github.com/UAlbanyArchives/mailbag/blob/develop/mailbag/derivatives/html.py#L33-L34